CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+Please, feel free to send your suggestions concerning any library issue, documentation and README typos, [Sample](./Sample) improvements, and any other enhancement. **We really appreciate your participation!**
+
+### How to ask a question
+
+Before asking a question, please, look through [README](./README.md) and [Sample application](./Sample). Then take a look into [already asked GitHub Issues](https://github.com/handsomecode/InteractiveSideMenu/issues) (even closed). If you still not found an answer, feel free to [open an issue](https://github.com/handsomecode/InteractiveSideMenu/issues/new) and ask us your question. It'd be great to mark it with prepared `question` label.
+
+### How to report a bug
+
+The standard way to report a bug is [GitHub Issues](https://github.com/handsomecode/InteractiveSideMenu/issues).
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+- **Use a clear and descriptive title** for the issue to identify the problem.
+- **Describe the exact steps which reproduce the problem** in as many details as possible.
+- **Provide specific examples** to demonstrate these steps. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use Markdown code blocks.
+- **Describe the behavior you observed** after following these steps and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected** to see instead and why.
+- **Include screenshots and animated GIFs** which show you following the described steps and clearly demonstrate the problem.
+- **If you're reporting that library crashed**, include a crash report with a stack trace.
+
+Include details about your configuration and environment:
+
+- **Which version of InteractiveSideMenu** are you using?
+- Do you test on the **Simulator** or a **Real device**?
+- **iOS version** that you test with.
+- **XCode version** if it could be helpful in your case.
+
+### How to request an enhancement
+
+**Enhancements** are features that you might like to suggest to a project, but aren't necessarily bugs/problems with the existing code. There is *a label* for enhancments in [Github's Issues](https://github.com/handsomecode/InteractiveSideMenu/issues), so you can tag issues as `enhancement`, and thereby allow us to prioritize issues/bugs reported to the project.
+
+
+### How to say us about your application
+
+If you are using **InteractiveSideMenu** in your project that is successfully published to AppStore, [send the link](https://github.com/handsomecode/InteractiveSideMenu/issues/new) to us and we'll add you to the list with applications, used our library. 
+
+
+### How to submit changes
+
+When contributing to this repository, please, first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
```

Comment:
In my view, this library is not so popular to advise asking questions on StackOverflow. Issues is enough for now.
What about emailing - this is in the case, when contributor doesn't want to discuss his idea publicly. Maybe, he is going to propose some secret idea or simply feels more confident to collaborate personally. And, specific mail is mentioned in this doc and in the code of conduct. Maybe, it makes sense to specify it again.

I agree that it complicates managing. But don't think that it could be a problem at this stage. Anyway, we can update this rule at any time if see fit. I'd like to keep as much entry points as possible for now.